pom.xml

```diff
@@ -38,8 +38,13 @@
       <artifactId>jetty-annotations</artifactId>
       <version>${jetty.version}</version>
     </dependency>
+    <!--DataStore-->
+    <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-datastore</artifactId>
+        <version>1.104.0</version>
```

Comment:
It probably doesn't matter, but there is a newer version available: [1.106.0](https://mvnrepository.com/artifact/com.google.cloud/google-cloud-datastore)

Reply:
Thanks for pointing it out! I think it would be better to stick with the old one to follow what we learned on the tutorials, and to avoid any weird changes (if there are any) we're not aware of.